wolfcrypt/src/ecc.c

```diff
@@ -5815,7 +5819,9 @@ static int _ecc_make_key_ex(WC_RNG* rng, int keysize, ecc_key* key,
         if (err == MP_OKAY) {
             err = ecc_make_pub_ex(key, curve, NULL, rng);
         }
-
+        else {
```

Comment:
Confusing move up.

Reply:
Same; I cleaned up that whole section regarding the verbose error messages.